docs/en/concepts-and-designs/lal.md

```diff
@@ -0,0 +1,191 @@
+# Log Analysis Language
+
+Log Analysis Language (LAL) in SkyWalking is essentially a Domain-Specific Language (DSL) to analyze logs. You can use
+LAL to parse, extract, filter, analyze, and save the logs, as well as collaborate the logs with traces, and metrics.
+
+Check the [`default.yaml`](../../../oap-server/server-bootstrap/src/main/resources/lal/default.yaml) for examples.
+
+## Filter
+
+A filter is a group of [parser](#parser), [extractor](#extractor) and [sink](#sink). Users can use one or more filters
+to organize their processing logics. Every piece of log will be sent to all filters in an LAL rule. The piece of log sent into the filter is available as property `log` in the LAL, therefore you can access the log level via `log.level`.
+
+### Parser
+
+Parsers are responsible for parsing the raw logs into structured data in SkyWalking for further processing. There are 3
+types of parsers at the moment, namely `json`, `yaml`, and `text`.
```

Comment:
JSON  = JavaScript Object Notation, should be upper case all the time.

Reply:
> JSON = JavaScript Object Notation, should be upper case all the time.

do you prefer something like 


```groovy
filter {
  JSON {
  }
  // or
  YAML {
  }
}
```

over 

```groovy
filter {
  json {
  }
  // or
  yaml {
  }
}
```

?

I know the abbreviation of JSON and YAML, sometimes when they serve as field / method / file names, IMHO it's ok to lowercase them. However I can uppercase them if you like.

Reply:
OK, let's keep them in lower case. I misunderstand your words.